_commands/basics/du.md

```diff
@@ -3,22 +3,79 @@
 
 du
 ---
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`du`, or _disk usage_, reports the sizes of directories, including of all of their contents and the sizes of individual files, and the sizes of each directory in the directory tree.
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ du [options] [directories and/or files]
+$ du
+$ du home/eecs398/hw1
+$ du -h
+$ du -sh
+$ du -a
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+#### `du`
+~~~ bash
+$ du
+$ 236 ./hw1
```

Comment:
This and the next couple lines are the output, correct? If so, the leading `$` can be removed.